src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  static fullHealthByDefault = 100;
+  constructor(name, health = Animal.fullHealthByDefault) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  biteDamage = 50;
```

Comment:
All constants you can move to the upper scope or separate file and named in format: const BITE_DAMAGE. ( for future)